vllm/compilation/inductor_pass.py

```diff
@@ -1,27 +1,58 @@
 # SPDX-License-Identifier: Apache-2.0
 
 import hashlib
+import importlib.metadata
 import inspect
+import json
 import types
 from abc import ABC, abstractmethod
-from typing import Any, Callable, Optional, Union
+from typing import Any, Callable, Dict, Optional, Union
 
 import torch
+from packaging.version import Version
 from torch import fx
 
+if Version(importlib.metadata.version('torch')) >= Version("2.6"):
+    from torch._inductor.custom_graph_pass import CustomGraphPass
+else:
+    # CustomGraphPass is not present in 2.5 or lower,
+    # and custom passes are pickled when determining the caching key.
+    # Declare CustomGraphPass from 2.6 and add pickling support.
+    class CustomGraphPass(ABC):  # noqa (redefinition)
```

Comment:
Shall we try naming this as something like OldCustomGraphPass? And import as CustomGraphPass?